src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -7,4 +7,5 @@
   <include file="changes/create-consortium-table.xml" relativeToChangelogFile="true"/>
   <include file="changes/create-tenant-table.xml" relativeToChangelogFile="true"/>
   <include file="changes/create-user-tenant-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/db.changelog-1.2.0.xml" relativeToChangelogFile="true"/>
```

Comment:
should be renamed to something like that: change-unique-index-in-user-tenant-table

Reply:
fixed